kolibri/plugins/user/views.py

```diff
@@ -1,18 +1,8 @@
-from __future__ import absolute_import, print_function, unicode_literals
-from django.views.generic.base import TemplateView
-from kolibri.auth.models import Facility
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
@rtibbles why are these here?

Reply:
Because they are not on one line any more (we had a change in python linter that now splits imports into repeated, but multiline).